docs/asdf/extending/extensions.rst

```diff
@@ -154,6 +154,26 @@ Tag handles can be defined in the ``yaml_tag_handles`` property of an extension:
         extension_uri = "asdf://example.com/example-project/extensions/foo-1.0.0"
         yaml_tag_handles = {"!example!": "asdf://example.com/example-project/tags/"}
 
+Additional schema validators
+----------------------------
+
+Schema validators implement the `Validator` interface
+and are included in an extension via the ``validators`` property:
+
+.. code-block:: python
+
+    from asdf.extension import Extension, Validator
+
+
+    class FooValidator(Validator):
+        # ...
+        pass
```

Comment:
```suggestion
    class FooValidator(Validator):
        schema_property = "bar"
        tags = ["asdf://example.com/example-project/tags/foo-1.0.0"]
        
        def validate(self, value, node, schema):
            if value != "baz":
                yield ValidationError("bar is not baz for foo object")
```
Should we flush this out to a complete example or link to the other example in the docs?

Reply:
Thanks!